src/Illuminate/Filesystem/Filesystem.php

```diff
@@ -38,7 +39,7 @@ public function exists($path)
      */
     public function missing($path)
     {
-        return ! $this->exists($path);
```

Comment:
Even though, StyleCI will take care of this when merging [as stated in the docs](https://laravel.com/docs/contributions#styleci) it's perhaps a little distracting, having formatting-only changes alongside logic additions

Reply:
Yeah you're right. My intelephense was what did the auto-formatting and I didn't notice

Reply:
We've all been there 😉